pkg/cloudprovider/providers/rackspace/rackspace.go

```diff
@@ -660,3 +660,21 @@ func (rs *Rackspace) DiskIsAttached(diskName, instanceID string) (bool, error) {
 	}
 	return false, nil
 }
+
+// query if a list volumes are attached to a compute instance
+func (rs *Rackspace) DisksAreAttached(diskNames []string, instanceID string) ([]bool, error) {
```

Comment:
@anguslees, @jamiehannaford, @jrperritt @mkulke: can one of you verify the rackspace changes by EOD?